tests/wcs1.py

```diff
@@ -467,17 +481,20 @@ def makeWcs(crPixPos, crValDeg, projection):
 
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
+@unittest.skipIf(afwdataDir is None, "afwdata not setup")
 class WCSTestCaseCFHT(unittest.TestCase):
     """A test case for WCS"""
 
     def setUp(self):
-        path = InputImagePath + ".fits"
-        self.metadata = afwImage.readMetadata(path)
-        self.wcs = afwImage.makeWcs(self.metadata)
+        if afwdataDir is not None:
+            path = InputImagePath + ".fits"
+            self.metadata = afwImage.readMetadata(path)
+            self.wcs = afwImage.makeWcs(self.metadata)
 
     def tearDown(self):
-        del self.wcs
-        del self.metadata
+        if afwdataDir is not None:
+            del self.wcs
```

Comment:
I'd be inclined to be explicit about calling del on `self.wcs` if `self.wcs` is an attribute and not None.